src/pages/networks/forms/NetworkDefaultACLReadOnly.tsx

```diff
@@ -0,0 +1,51 @@
+import type { FormikProps } from "formik";
+import type { FC } from "react";
+import type { NetworkFormValues } from "./NetworkForm";
+import { Icon } from "@canonical/react-components";
+
+const NetworkDefaultACLReadOnly: FC<{
+  formik: FormikProps<NetworkFormValues>;
+}> = ({ formik }) => {
+  const egress = formik.values.security_acls_default_egress;
+  const ingress = formik.values.security_acls_default_ingress;
+
+  const conjugateAction = (action: string) => {
+    return action === "drop" ? "dropped" : action + "ed";
+  };
+
+  if (formik.values.security_acls.length === 0) {
+    //When no value is set, LXD ACLs default to "reject"
+    return (
+      <div className="">
+        When no ACL rule matches, <b>egress</b> and <b>ingress</b> traffic will
+        be <code>Rejected.</code>
+      </div>
+    );
+  } else if (egress === ingress) {
+    return (
+      <div className="">
+        When no ACL rule matches, <b>egress</b> and <b>ingress</b> traffic will
+        be <code>{conjugateAction(egress ?? "")}</code>
+      </div>
+    );
+  } else {
```

Comment:
I think we should remove these cases and always show the split view with icons. Then the user doesn't have to think when going into edit mode to understand a very different representation of the information. Currently, the representation changes. With removing this, the representation becomes symmetrical. So a user learns this once and can reuse that on edit.